scripts/vm.sh

```diff
@@ -90,6 +76,12 @@ EOF
 	qemu-img create -f qcow2 -F qcow2 -b "$DEBIAN_QCOW2" "$OVERLAY_QCOW2" "$OVERLAY_DISK_SIZE"
 
 	log "Creating virtual machine"
+
+	if is_gentoo; then
+		mkdir /var/lib/libvirt/shared/debian12-pydokku
+		mkdir /var/lib/libvirt/boot
+	fi
+
```

Comment:
I didn't understand why those lines are needed, since there's `mkdir -p "$SHARED_FOLDER"` inside `create_shared_folder` already (which does the same as the first line inside the `if`) and `/var/lib/libvirt/boot` is not used at all.
I suggest you trying to completely remove this `if`, completely remove the VM (`virt-manager` could help) and running the `make vm-create` without them to check if this is really required.
```suggestion
```